native/src/components/base/InputSection.tsx

```diff
@@ -95,6 +95,7 @@ const InputSection = ({
         returnKeyType='done'
         blurOnSubmit
         role='searchbox'
+        testID={title ?? value}
```

Comment:
:upside_down_face: Testing by `testID` should be avoided if possible. Have you tried querying by role/placeholder or using `.nearest()` on the title?

Reply:
I agree but I didn't manage to get it to work with any other ways I tried, including placeholder, title, or role. I don't really like going by role and then the nth of that because that makes the test pretty fragile.